packages/gatsby-plugin-sitemap/src/internals.js

```diff
@@ -20,29 +20,27 @@ export const defaultOptions = {
           siteUrl
         }
       }
-      allMarkdownRemark(
-        limit: 1000,
+      
+      allSitePage(
+        limit: 1000000000,
```

Comment:
hmm... it's GraphQL complaining. Never mind then. Maybe a PR into GraphQL to allow `Infinity` and `-Infinity` is in order at some point? Not sure if removing limit works for GQL.